sway/input/seatop_default.c

```diff
@@ -489,6 +493,7 @@ static void handle_button(struct sway_seat *seat, uint32_t time_msec,
 		node = seat_get_focus_inactive(seat, &cont->node);
 		seat_set_focus(seat, node);
 		seat_pointer_notify_button(seat, time_msec, button, state);
+		transaction_commit_dirty();
```

Comment:
Nit: In the other places, the order is focus, commit, pointer notify, while here it's focus, pointer notify, commit. It would be good to align the behavior.

Reply:
I guess in my head it was "do everything with seat_* and then commit the transaction" :sweat_smile: 
Fixed.